### DISCUSSION TITLE
Help charging my MacBook

### DISCUSSION THREAD

**Discussion by felixvklein** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1175] [General]
Hi guys

so I currently have a MacBook Pro 14 M1 and I use pretty much every hour of the day. From 10am till 10pm. I downloaded AlDente last week and I'm only charging to 70%. 

The concern that I have is, I'm charging twice a day. That never happened when I had my Macbook at 100%. 

My question is, is that good or not? Do I do the right thing with charging twice but not till 100? Because it just seems so unlogical charging till 70% twice a day.

Thanks for the help

FELIX

**Comment by MatthiasKerbl** [comment]
Hi @felixvklein,

When it comes to long-term battery health, it is best to leave your MacBook plugged in and use a charge limit of about 70-80% depending on what's best for you. However, if you don't always have a plug close. Studies show that it is better to charge twice between 30-80% instead of once between 0-100%.

Best,
Matthias

**Reply by felixvklein** [reply]
Thanks once again! So even if I hit 80%, I should still use my charger to charge my MacBook?

- Felix

**Reply by MatthiasKerbl** [reply]
Yes, just leave it plugged in so that your MacBook is drawing energy from the charger and bypassing the battery.

Best,
Matthias

**Reply by felixvklein** [reply]
hey thanks Matthias. 

really appreciate it. 

I was on 80% battery yesterday and hopped on a zoom call for 40 minutes. I lost 14% battery. I went from 66 to 20 in 4hours.. I'm using zoom all the time and now I'm afraid I'm fucking up my battery if I just keep in plugged all the time and work with 80% battery. 

my battery power is 96%. any suggestions/similar cases?

**Reply by MatthiasKerbl** [reply]
Research studies indicate that leaving a battery at around 80% is better than cycling it between 20 and 80% for slow battery aging. Therefore, if you have a charger close, leave it plugged in and set a charge limit.

Best,
Matthias

**Reply by felixvklein** [reply]
so even though I'm on zoom calls. I should charge till 80% and then keep it plugged in and keep aldente open? Is it normal to lose 44% in 4 hours? Seems a lot to me. Just scared I'M damaging my battery more.

What if I take the MacBook to move. Do I charge till 100% or still 80%?

**Reply by MatthiasKerbl** [reply]
Yes, you should just leave it plugged in AlDente running to keep the charge limit. Losing 44% in four hours can be completely normal, just depends on your setup and usage. If you are on the go and you need the full battery charge to 100%, if 80% is enough for the day, then don't charge to 100%.

Best,
Matthias

**Comment by felixvklein** [comment]
much appreciated it. thanks Matthias
